README.md

```diff
@@ -62,6 +62,31 @@ Follow linked crate name for detailed status. Please note that all crates follow
 
 [semver]: https://semver.org
 
+### Pipeline Integration
+
+Some CI/CD pipelines leverage repository cloning. Below is a copy-paste-able example to build docker images for such workflows.
+
+#### Pipeline Integration (recommended)
+
+Build an image without a target and then copy the binaries into your local image
```

Comment:
I don't get that - could you describe that without assuming too much docker knowledge?

Reply:
I am trying to say that I don't understand the paragraph, and that it should be adjusted to fit people who don't do that every day. Ideally, no prior knowledge about this usually works is assumed.